package.json

```diff
@@ -36,6 +39,7 @@
     "eslint-plugin-promise": "^6.1.1",
     "express": "^4.17.1",
     "jest": "^29.5.0",
+    "jsdocs": "^1.0.0",
```

Comment:
This is not the JSDoc package.

`npm install --save-dev jsdoc`

This is the package `jsdoc`

They seem to be on v3.5.5.

We also don't need this package direclty because jsdoc comes with the template